.github/ISSUE_TEMPLATE/intent-to-implement--i2i-.md

```diff
@@ -19,11 +19,17 @@ If you haven't already done so, sign the Contributor License Agreement (CLA) as
 <!--
 Provide a brief description of the feature/change you are planning on implementing.
 -->
+Link to PRD: 
+<!--
+Add a link to the PRD (make sure it is accessible by anyone) here.
+-->
 
-## Design document
+## Design documents
 <!--
 Provide a link to your design document once you have one. You do not need a design document to file this I2I.
 -->
+- Technical Design Document
```

Comment:
Here too, it would be useful to have pointers to what kind of document, exactly, is expected (examples and templates would be useful).

Reply:
Taken a stab at this inline.

Reply:
I'm slightly confused. A Design document is itself a Github issue?

The bit.ly/amp-autocomplete-ux link requires a slack account to view. Think you could host it somewhere else?

Reply:
I think there would be value in moving towards a single format for those and be more prescriptive here. But let's discuss that in #24132.